client/src/components/Form/BlogsInputs/BlogsTitleInput.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import FormInput from '../FormInput';
+
+const BlogsTitleInput = (props) => (
+    <FormInput
+    required
+    placeholder="Enter Title for your blog"
+    type="text"
+    label="Enter Title"
+    id="blogsTitle"
+    {...props}
+  />
```

Comment:
same with this, if FormInput is abstracted what is the use of creating another component like this.

Reply:
Alright will change this